spec/views/items/_embargo_form.html.erb_spec.rb

```diff
@@ -1,13 +1,16 @@
 require 'spec_helper'
 
 RSpec.describe 'items/_embargo_form.html.erb' do
-  let(:current_user) do
-    double('current_user', is_admin: true)
+  let(:current_user) { mock_user(is_admin?: true) }
+
+  before do
+    allow(view).to receive(:current_user).and_return(current_user)
+    allow(controller).to receive(:current_user).and_return(current_user)
```

Comment:
Sorry, and having said that, it's obvious that change is in the wrong PR. I'll try to move it to #596.